main/parsers.h

```diff
@@ -51,6 +51,7 @@
 	PythonParser, \
 	RexxParser, \
 	RubyParser, \
+	RestParser, \
```

Comment:
> That would apply to classes and method names then, too?!

If you mean things like RestKinds[] and findRestTags()... Yes, it would.

> Thought about calling it RstParser
> Maybe ReST could be used instead?

I have no strong option on "Rst" or "ReST", just a slight preference for Rst (but mainly because it looks better in [CamelCasing](https://en.wikipedia.org/wiki/CamelCase))

* "Rst" is the most common used file extension. And note the URL of the homepage: http://docutils.sourceforge.net/rst.html
* "ReST" matches the official abbrev spelling best.